dev/values/validators/single.yml

```diff
@@ -1,6 +1,12 @@
 global:
   namespaceOverride: astria-dev-cluster
 
+images:
+  sequencer:
+    repo: ghcr.io/astriaorg/sequencer
+    tag: "0.14.0"
+    devTag: sha-b4c5d1c
```

Comment:
Should this be a specific devTag sha instead of latest? can add a new `single-ics20.yml` for now instead or manually change when using.

Reply:
It should be latest, but the image hasn't been built yet because the fix is in this pr